<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arap_widget</class>
 <widget class="QWidget" name="arap_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>As-rigid-as-possible Deformation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Anchor type</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="manualAnchor">
        <property name="text">
         <string>Manually selected</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorModes</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="distanceAnchor">
        <property name="text">
         <string>Distance</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorModes</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="distanceSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of iterations</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="numIterations">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="controlButton">
       <property name="text">
        <string>Control points</string>
       </property>
       <property name="icon">
        <iconset resource="surfacemesh_mode_arapdeform.qrc">
         <normaloff>:/images/control_points.png</normaloff>:/images/control_points.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="anchorButton">
       <property name="text">
        <string>Anchor points</string>
       </property>
       <property name="icon">
        <iconset resource="surfacemesh_mode_arapdeform.qrc">
         <normaloff>:/images/anchor_points.png</normaloff>:/images/anchor_points.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deformButton">
       <property name="text">
        <string>Deform</string>
       </property>
       <property name="icon">
        <iconset resource="surfacemesh_mode_arapdeform.qrc">
         <normaloff>:/images/deform.png</normaloff>:/images/deform.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="surfacemesh_mode_arapdeform.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="anchorModes"/>
 </buttongroups>
</ui>
